Chapters/6_Qualified_identifiers.md

```diff
@@ -86,18 +86,34 @@ indicates that the content seen previously with the function `generate_intput_st
 been seen in the Git repository at `https://gitorious.org/ocamlp3l/ocamlp3l_cvs.git`, when
 its full state had the SWHID core identifier `swh:1:snp:d7f1b9eb7ccb596c2622c4780febaa02549830f9`, and that it is named `simplefarm.ml` in the directory `Simplefarm` contained in the directory `Examples` contained in the root directory associated to the revision with core SWHID `swh:1:rev:2db189928c94d62a3b4757b3eec68f0a4d4113f0`.
 
+### 6.3 Comparing qualified SWHIDs
 
-### 6.3 Recommendations
+One can determine whether two software artifacts are identical (bit by bit) by
+comparing their core SWHIDs, ignoring all qualifiers. If the core SWHIDs match,
```

Comment:
Minor: "match" is not defined here and people might wonder what it means.
I propose: "match" → "are equal"
("equal" instead of "identical" just to avoid the repetition with the sentence that follows)